### ISSUE TITLE
How to fit a multi component SNAP potential using the Python interface? 

### ISSUE THREAD

**Issue by jan-janssen** [FitSNAP/FitSNAP#240]
My settings object: 
```
settings = \
{
"BISPECTRUM":
    {
    ...
    "twojmax": [8, 8],
   ...
    },
...
}
```

When I initialise FitSNAP using `snap = FitSnap(settings, comm=comm, arglist=["--overwrite"])` I get the following error: 
```
File ~/mambaforge/lib/python3.10/site-packages/fitsnap3lib/io/sections/calculator_sections/bispectrum.py:82, in Bispectrum._generate_b_list(self)
     80 for atype in range(self.numtypes):
     81     i = 0
---> 82     for j1 in range(int(max(self.twojmax)) + 1):
     83         for j2 in range(j1 + 1):
     84             for j in range(abs(j1 - j2), min(int(max(self.twojmax)), j1 + j2) + 1, 2):

ValueError: invalid literal for int() with base 10: '[8,'
```

It seems to be related to specifying the `twojmax` as list, when I specify it as string `"twojmax":  "8 8"` everything works fine, but it is not very intuitive for new users.

**Comment by jan-janssen** [comment]
I had some more issues with setting:
```
    "wj": "1.0 1.0",
    "radelem": "0.5 0.5",
    "type": "Li Al",
```

Still in the end, I am able to process the configuration:
```
snap.process_configs(data)
```
This raises a couple of `WARNING: Configuration has no SNAP neighbors` messages but I presume this is just related to not all species appearing in each configuration. 

Afterwards I call:
```
snap.solver.perform_fit()
coeffs = snap.solver.fit
coeffs
```

This gives me the coefficients for one potential, but I expected three potentials `1 1`, `2 2` and `1 2`.

**Comment by rohskopf** [comment]
Specifying multielement settings as a string is just a convenient way to have backward compatibility with configparser utility. This can be changed but should at least be documented, so will leave this open until then.

**Comment by rohskopf** [comment]
> This gives me the coefficients for one potential, but I expected three potentials 1 1, 2 2 and 1 2.

Yeah this does sound buggy. There should be coeffs for all types. The WBe example has some configs that are only one type, but gives coeff like normal. It uses input script instead of settings duct, but technically all examples use the library now.
